hw/ip/aes/dv/aes_sim_cfg.hjson

```diff
@@ -46,6 +46,8 @@
   uvm_test: aes_base_test
   uvm_test_seq: aes_wake_up_vseq
 
+    xcelium_cov_refine_files: ["{proj_root}/hw/ip/aes/dv/cov/aes_unr.vRefine"]
```

Comment:
is this still needed? Looks like you have another PR to add aes exclusion.

Reply:
@weicai - this is a draft PR that will never be checked in - it was just a fast way to share the work!
so ignore anything that is not dvsim related.

Reply:
sure, sounds good.